packages/contract_templates/partials/callAsync.handlebars

```diff
@@ -15,6 +15,7 @@ async callAsync(
     const callDataWithDefaults = await BaseContract._applyDefaultsToTxDataAsync(
         {
             to: self.address,
+            ...callData,
```

Comment:
@fabioberger @lgedeon - I added this so I could override the `from` address in `callAsync`.